src/Serilog.Settings.Configuration/Settings/Configuration/Assemblies/DependencyContextAssemblyFinder.cs

```diff
@@ -27,7 +27,7 @@ where IsCaseInsensitiveMatch(assemblyName.Name, nameToFind)
             
             static bool IsReferencingSerilog(Library library)
             {
-                return library.Dependencies.Any(dependency => dependency.Name.Equals("serilog", StringComparison.OrdinalIgnoreCase));
+                return library.Dependencies.Any(dependency => dependency.Name.IndexOf("serilog", StringComparison.OrdinalIgnoreCase) >= 0);
```

Comment:
not 100% sure about the impact here 🤔 

Could it be problematic if there is a package like `SomeRandomThingWithTheNameSerilogInIt` ? should we stick to "starts with" serilog maybe ?

Reply:
makes sense 👍